package.json

```diff
@@ -38,8 +38,8 @@
         "@babel/plugin-transform-flow-strip-types": "^7.4.4",
         "@babel/preset-env": "^7.5.5",
         "@babel/preset-react": "^7.0.0",
-        "@ckeditor/ckeditor5-dev-utils": "^12.0.1",
-        "@ckeditor/ckeditor5-theme-lark": "^18.0.0",
+        "@ckeditor/ckeditor5-dev-utils": "^23.5.1",
+        "@ckeditor/ckeditor5-theme-lark": "^23.0.0",
```

Comment:
you need to adjust the `package.json` in your project if you build the javascript by yourself, right? cas we add something about this to the `UPGRADE.md`? this caused a lot of problems in our Slack channel last time